body.tex

```diff
@@ -74,20 +117,22 @@ \subsubsection{Drawbacks}
 
 
-\subsubsection{Benefits}
+\subsection{Benefits}
 \label{sec:rb_benefits}
 
-Although as discussed this is just a sub-optimal learning-based approach to this problem, it still has the advantage of \emph{interpretability}; the generated output might be too contamined with false alarms and (at the same time) missed true transients, but the process is fully transparent to the downstream users of the alerts and they have the possibility to develop their own post-processing blocks per need.
+Although as discussed this is just a sub-optimal learning-based approach to this problem, it still has the advantage of \emph{interpretability}; the generated output might be too contaminated with false alarms and (at the same time) missed true transients, but the process is fully transparent to the downstream users of the alerts and they have the possibility to develop their own post-processing blocks per need.
 
 
-\subsubsection{What will be learned?}
-Since the temporal behaviour of the candidates fed to this module is assumed, it is foreseeable that the neural network will eventually learn a concept close to ``PSF-ness'', true to its title; real/bogus classification.
+\subsection{What will be learned?}
+Since the temporal behavior of the candidates fed to this module is assumed, it is foreseeable that the neural network will eventually learn a concept close to ``PSF-ness'', true to its title; real/bogus classification.
 
-\subsubsection{Modified 3D Real/Bogus Classifier}
-In this implementation the classifier has access to the template, science image pairs along with the diff image. This allows the network to make use of the temporal behavior of the source too.
-Note however that still the difference image has a key role in the process, as the images are cropped around the sources detected in the difference image.
+\section{Modified 3D Real/Bogus Classifier}
+\label{sec:modified}
+In this implementation, we try to feed the network with a glimpse of the temporal behaviour of the candidate, by passing the template and science image pairs along with the diff image. This allows the network to make use of the temporal behavior of the source too.
+Note however that still the difference image plays a key role in the process, as the images are cropped around the sources detected in the difference image. This means that the modification only aims for clealiness of the detections, by passing more information about the existing candidates (diaSources), but cannot affect the completeness of detections, by definition.
 
-\subsection{End-to-End Simultaneous Localization and Classification (~TransiNet)}
+\section{Future Work}
+\subsection{End-to-End Simultaneous Localization and Classification (TransiNet)}
```

Comment:
```suggestion
\subsection{End-to-End Simultaneous Identification and Localization (TransiNet)}
```

"Classification" has existing other meanings to science users so it would be useful to avoid it.

Reply:
This is the terminology conflict (between Vision and Astro) where we should be very careful not to be wrong in either of the areas. What does "Classification" exactly mean in astro? 
Or put differently, what is a solution to "real transient or not?" called in astro?

Reply:
"Classification" in this context in astro is typically meant to mean determining what the astrophysical origin of the emission is (e.g., a supernova, an AGN, etc.).

maybe "simultaneous identification and artifact filtering"?  it's not perfect though.